src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form 
```

Comment:
create common classes for forms, inputs and buttons, and move there all repetitive styles, unique ones live for different styles